src/ballet/sha256/fd_sha256.h

```diff
@@ -61,6 +62,9 @@ fd_sha256_align( void );
 FD_FN_CONST ulong
 fd_sha256_footprint( void );
 
+// Synchronization functions
```

Comment:
"Synchronization" doesn't strike me as the right adjective, but I don't know what other parts of the codebase use